js/templates/trackerlist-truncated.es6.js

```diff
@@ -6,13 +6,13 @@ module.exports = function () {
     if (this.model.companyListMap && this.model.companyListMap.length > 0) {
 
         return bel`<section class="top-blocked card">
-            <h3 class="menu-title">Top blocked over time</h3>
+            <h3 class="menu-title">Top blocked companies over time</h3>
             <ul class="menu-list top-blocked__list">
                 ${trackerListItems(this.model.companyListMap)}
                 <li class="top-blocked__li top-blocked__li--see-all border--top">
                     <a href="#" class="link-secondary js-top-blocked-see-all">
                         <span class="icon icon__arrow pull-right"></span>
-                        See all
+                        See tracker request counts
```

Comment:
Seems weird to me to have "See tracker request counts" twice on the popup but going to separate views.